spec/booking_system_spec.rb

```diff
@@ -0,0 +1,58 @@
+require_relative 'spec_helper'
+require_relative '../lib/booking_system'
+
+describe "what the hotel booking system does" do
+  before do
+    @room_number = 12
+    @check_in = Date.new(2018, 8, 24)
+    @check_out = @check_in + 5
+    @booking_system = Hotel::BookingSystem.new
+  end
+
+  it "create a class for the booking system" do
+    @booking_system.must_be_kind_of Hotel::BookingSystem
+  end
```

Comment:
You should break these tests up a little. I typically have one big `describe` for the whole class, and then one sub-`describe` for each instance method.